protected/widgets/menu/assets/menu.css

```diff
@@ -23,61 +23,68 @@
     padding-left: 2em;
 }
 
-
 .menu ul {
     margin: 0;
 }
 
 .menu {
-    overflow: hidden;
```

Comment:
Why remove this line

Reply:
I though it would be better to have the variables declaration at the beginning. Didn't think it would appear in the commit. I'll try not to do that again